datumaro/plugins/splitter.py

```diff
@@ -0,0 +1,480 @@
+# Copyright (C) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import logging as log
+import numpy as np
+
+from datumaro.components.dataset import Dataset
+from datumaro.components.extractor import (Transform, AnnotationType, \
+    DEFAULT_SUBSET_NAME)
+
+class TaskSpecificSplitter(Transform):
+    def __init__(self, dataset, seed):
```

Comment:
I see. 
I'll add the command line feature later for the other classes (SplitterforXXX) because I have not tested it yet.
Can I add the feature later, not in this pull request?

Reply:
Ok, no problems with this. Actually, they are added automatically, it's just some polishing and customization.